### PR TITLE
added m2r2 and redirected contribution file to sphinx doc file.

### PR DESCRIPTION
**Pull request by anistark**
Fix for issue #28

### CODE REVIEW

File: docs/source/requirements.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
+m2r2==0.2.7
```
Comment (goodwillcoding) [inline docs/source/requirements.txt]: docs/source/requirements.txt should be deleted (it was left over cruft)
instead use `poetry add -D m2r2` which will add the m2r2 package to poetry. Make sure to run `poetry lock` after the add

File: README.rst [hunk 2]
```diff
@@ -50,4 +50,4 @@ Install
 Contributing
 ------------
 
-Contributions are welcome, and more information is available in the `contributing guide <CONTRIBUTING.md>`_.
+Contributions are welcome, and more information is available in the `contributing guide <contributing.html>`_.
```
Comment (goodwillcoding) [inline README.rst]: does contributing.html work? As in were you able to run `make docs` and see it there

Comment (nkantar) [reply README.rst]: I think that link should probably point to what will ultimately be the published page: https://parsenvy.readthedocs.io/en/latest/contributing.html

The current value breaks RTD, and this change breaks GitHub, but linking to the published page should fix both.

File: docs/source/requirements.txt (hunk 1 above)
Comment (nkantar) [reply docs/source/requirements.txt]: @goodwillcoding will RTD pick up the Poetry config? IIRC RTD relies on requirements.txt.

**Review by nkantar — CHANGES_REQUESTED** [review]

File: docs/source/requirements.txt (hunk 1 above)
Comment (nkantar) [reply docs/source/requirements.txt]: `docs/source/requirements.txt` is now deleted from main. Here are the steps to add this dependency:

1. run `poetry add m2r2 --dev`
2. edit `pyproject.toml`, to move it under the `# docs` section of `[tool.poetry.dev-dependencies]`
3. add its name to the `[tool.poetry.extras]` as well
4. commit `pyproject.toml` and `poetry.lock`

Comment (nkantar) [reply docs/source/requirements.txt]: Disregard—not needed since the doc is rST. :)

**Pull request merged**